src/app/api/fetch_user/route.ts

```diff
@@ -0,0 +1,32 @@
+"use server"
```

Comment:
is it required? Isn't this the default??

Reply:
yes by default it is but i saw in some videos to provide strictness to the api we use it but yeah it's optional. changes done